NextGenConnect/JSON to HL7v2.xml

```diff
@@ -253,21 +244,40 @@ var location = msg[&apos;entry&apos;].filter(function (entry){
     return entry[&apos;resource&apos;][&apos;resourceType&apos;] === &quot;Location&quot;;
 })[0][&apos;resource&apos;];
 
+/* Need to add these to the bundle
+var organization = msg[&apos;entry&apos;].filter(function (entry){
+    return entry[&apos;resource&apos;][&apos;resourceType&apos;] === &quot;Organization&quot;;
+})[0][&apos;resource&apos;];
+
+var actor = msg[&apos;entry&apos;].filter(function (entry){
+    return entry[&apos;resource&apos;][&apos;resourceType&apos;] === &quot;Practioner&quot;;
+})[0][&apos;resource&apos;];
+*/
 
 //RXA-1 Must be 0
 tmp[&apos;RXA&apos;][&apos;RXA.1&apos;] = 0;
 
 //RXA-2 Must be 1
 tmp[&apos;RXA&apos;][&apos;RXA.2&apos;] = 1;
 
-//RXA-3 Data of Administration -- Need to worry if time component stored 
-//tmp[&apos;RXA&apos;][&apos;RXA.3&apos;] = immunization[&apos;occurenceDateTime&apos;].replace(&apos;-&apos;,&apos;&apos;); // commented out due to undefined
+//RXA-3 Data of Administrationmsg[&apos;entry&apos;][0][&apos;resource&apos;][&apos;occurrenceDateTime&apos;]
+
+var occurenceDate = immunization[&apos;occurrenceDateTime&apos;];
+tmp[&apos;RXA&apos;][&apos;RXA.3&apos;] = occurenceDate.split(&apos;-&apos;).join(&apos;&apos;); 
 
-//RXA-5 Administered Code -- Needs to be CVX -- how are we converting what&apos;s stored in FHIR resource? Hardcode?
+//RXA-5 Administered Code -- // NDC code?
```

Comment:
Are we handling RXA-5 in this card? If not, is there a card in the backlog to deal with it?

Reply:
It is handled. WIR specification suggests we include the NDC code too (5.1.4-5.1.6) although it is not necessary. However the 5.1.1-5.1.3 have inputs.

Reply:
Where? I don't see it handled in any code that isn't commented out.

Reply:
The code is commented out because I need to add the references to the other resources to access the value or else it's undefined.

Reply:
See: *Need to add organization to bundle*

Reply:
This is fixed. However need to discuss the NDC code being added.